compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -566,7 +566,7 @@ instance Pretty DefDataType where
     DataEnum enums ->
       (keyword_ "enum" <-> lhsDoc) $$ nest 2 (vcat (map pPrintEnumCon enums))
     DataInterface ->
-      keyword_ "interface"
+      keyword_ "interface" <-> pPrint tcon
```

Comment:
not crucial, but I was very confused about the lines with just `interface` in the pdalfs